gymManagement/src/main/java/com/angef/gymManagement/dto/ManagementDTO.java

```diff
@@ -0,0 +1,53 @@
+package com.angef.gymManagement.dto;
+
+public class ManagementDTO {
+	    private boolean invoiced;
+	    private boolean access;
+	    
+	    private ManagementDTO() {
+	    }
+
+	    public boolean isInvoiced() {
+	        return invoiced;
+	    }
+
+	    
+	    public boolean hasAccess() {
+	        return access;
+	    }
+
+	    public static class Builder {
+	        private boolean invoiced;
+	        private boolean access;
+
+	        public Builder() {
+	        }
+
+	        public Builder withInvoiced(boolean invoiced) {
+	            this.invoiced = invoiced;
+	            return this;
+	        }
+
+	        public Builder withAccess(boolean access) {
+	            this.access = access;
+	            return this;
+	        }
+
+	        public ManagementDTO build() {
+	            ManagementDTO managementDTO = new ManagementDTO();
+	            managementDTO.invoiced = this.invoiced;
+	            managementDTO.access = this.access;
+	            return managementDTO;
+	        }
+	    }
```

Comment:
en este ManagementDTO build() se puede simplificar ya no necesitas crear un objeto de tipo ManagementDTO simplemente haces un return y creas el objeto pasandole la palabra reservada this y eso hará alusion a todos los campos sin necesidad de especificarlos. Fijate en el SubscriberDTO.